src/NSwag.Generation.AspNetCore/Processors/OperationParameterProcessor.cs

```diff
@@ -184,8 +184,8 @@ public bool Process(OperationProcessorContext operationProcessorContext)
 
                 if (operationParameter != null)
                 {
-                    operationParameter.Position = position;
                     position++;
+                    operationParameter.Position = position;
```

Comment:
What's the rationale for this change?

Reply:
Nothing, looks better to me :-)

Reply:
Is Position supposed to be 0-based or 1-based ?

Reply:
with `position = 1` above, that will make you start at 2, which I'm not sure is the wanted start number :) Did you mean to set `position = 0` instead ?

Reply:
Oh shit, yes, reverted.